distributed/dashboard/components/scheduler.py

```diff
@@ -35,7 +35,8 @@
 from bokeh.io import curdoc
 import dask
 from dask.utils import format_bytes, key_split
-from toolz import pipe
+from tlz import pipe
+from tlz.curried import map, concat, groupby
```

Comment:
Can use built in `map` function

Reply:
This is curried so idk if that would work here.

Reply:
Ah, yep you're right. I overlooked the `curried` part of this `import`